db/backup_download.go

```diff
@@ -102,6 +103,16 @@ func DownloadBackup(app, addon, backupID string, opts DownloadBackupOpts) error
 		return errgo.Notef(err, "fail to start download")
 	}
 	defer resp.Body.Close()
+
+	// Return an error if the api return 400
```

Comment:
The comment does not add anything that the code already says, moreover it is wrong

```suggestion
```